src/app/components/PortraitVideoCarousel/PortraitVideoPromo/index.tsx

```diff
@@ -0,0 +1,27 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { PortraitVideoPromoProps } from '#app/models/types/portraitVideo';
+import Image from '#app/components/Image';
+import Heading from '../../Heading';
+import styles from './index.styles';
+
+export default (item: PortraitVideoPromoProps) => {
+  const { images, headlines, onClick } = item;
+
+  const imageUrl = images?.[0]?.url;
+  const alt = images?.[0]?.altText || '';
+  const headline = headlines?.promoHeadline || '';
+
+  return (
+    <button type="button" onClick={onClick} css={styles.button}>
+      {imageUrl && (
+        <Image alt={alt} src={imageUrl} css={styles.image} lazyLoad />
+      )}
```

Comment:
Maybe more of a product decision but I feel like images shouldn't be optional for a carousel item. What would we show in its place, a placeholder of some kind?